Paco-Server/src/com/google/sampling/experiential/server/CloudSqlSearchServlet.java

```diff
@@ -122,6 +122,7 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
       resp.setContentType("text/plain");
```

Comment:
Is the output actually text/plain? Isn't it application/json?

Reply:
Looks like this comment was missed. I looked it up as well. In this case, the ContentType should indeed be "application/json".